src/Analyzer/CodeAnalyzer.cs

```diff
@@ -151,6 +151,7 @@ public ImmutableArray<DiagnosticAnalyzer> GetAnalyzers()
                     var xmlDoc = XDocument.Load(sonarConfigurationPath);
                     PatternIds = xmlDoc.Descendants("Rule").Select(e => e.Elements("Key").Single().Value)
                         .ToImmutableList();
+                    if(PatternIds.IsEmpty) return availableAnalyzers;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/codacy/codacy-sonar-csharp/pullRequest?prid=5517911)